cmd/peers.go

```diff
@@ -17,9 +17,8 @@ func NewPeersCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	o := &PeersOptions{IOStreams: ioStreams}
 	cmd := &cobra.Command{
 		Use:   "peers",
-		Short: "Commands for working with peers",
-		Long: `
-The ` + "`peers`" + ` commands allow you to interact with other peers on the Qri network.
+		Short: "commands for working with peers",
+		Long: `The ` + "`peers`" + ` commands allow you to interact with other peers on the Qri network.
```

Comment:
While we're here. `allow you to` is passive voice (not your change, I know). I'd prefer `The ` + "`peers`" + ` commands interact with other peers on the Qri network.`